configmap.go

```diff
@@ -74,3 +78,37 @@ func (test *Test) DeleteConfigMap(ConfigMap *v1.ConfigMap) {
 	err := test.deleteConfigMap(ConfigMap)
 	test.err(err)
 }
+
+// GetConfigMap returns a ConfigMap object if it exists.
+func (test *Test) GetConfigMap(ns, name string) *v1.ConfigMap {
```

Comment:
On second thought, sometimes we want to get an error, when we expect the resource is not there, so maybe hiding it is not the best idea. Will remove this and rather export the below func. Unless you have better idea on how to handle it?

Reply:
The idea was to use the default API to not have errors and add additional entry points that would have errors if need need to check them (say, ending those entrypoints with `Error`). But maybe that's overkill.